src/main/java/jenkins/plugins/mattermost/StandardMattermostService.java

```diff
@@ -29,7 +29,7 @@
 	public StandardMattermostService(String endpoint, String roomId, String icon) {
 		super();
 		this.endpoint = endpoint;
-		this.roomIds = roomId.split("[,; ]+");
+		this.roomIds = roomId.split("[,;]+");
```

Comment:
Right, I missed the `trim` part... Then it looks good to me :-)

I will add a note about (potential) configuration breaking [as detailed here](https://wiki.jenkins.io/display/JENKINS/Hint+on+retaining+backward+compatibility)